esi_leap/resource_objects/test_node.py

```diff
@@ -13,7 +13,7 @@
 from esi_leap.resource_objects import base
 
 
-class TestNode(base.ResourceObjectInterface):
+class FakeNode(base.ResourceObjectInterface):
```

Comment:
Not a huge deal, but would it make sense to rename the file `fake_node.py`?

Reply:
Sure, all done!